src/main/java/org/intellij/sdk/codesync/Constants.java

```diff
@@ -76,6 +77,10 @@ private Constants() {
 
     public static String SYNC_IGNORE_COMMENT = "# CodeSync won't sync the files in the .syncignore. It follows same format as .gitignore.";
 
+    public static String GLOBAL_LOCK_KEY = "global";
```

Comment:
Do we need this now?

Reply:
I left it for the sake of completeness. I will remove it to avoid confusion.